core/src/main/java/org/jdeferred/impl/AbstractPromise.java

```diff
@@ -18,17 +18,7 @@
 import java.util.List;
 import java.util.concurrent.CopyOnWriteArrayList;
 
-import org.jdeferred.AlwaysCallback;
-import org.jdeferred.DoneCallback;
-import org.jdeferred.DoneFilter;
-import org.jdeferred.DonePipe;
-import org.jdeferred.FailCallback;
-import org.jdeferred.FailFilter;
-import org.jdeferred.FailPipe;
-import org.jdeferred.ProgressCallback;
-import org.jdeferred.ProgressFilter;
-import org.jdeferred.ProgressPipe;
-import org.jdeferred.Promise;
+import org.jdeferred.*;
```

Comment:
not a fan of star imports. I think it would be better to revert back to an import statement per class.